### ISSUE TITLE
 Content Specific QA the 686 Form

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#7201]
## Goal
All content and fields should match exactly to what we have in the mockups ahead of a more official QA by Tze

## Considerations
- Should be done before e2e tests (#6049)
- [View Dependents punch list](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/ebenefits/features/view-update-dependents/research-design/text-revisions/view-dependents-punch-list-4-22.md)
- [Submit 686 punch list](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/ebenefits/features/view-update-dependents/research-design/text-revisions/686-punch-list-0420.md)

## Tasks
- On a screen by screen basis, match all components with what is displayed in the mockup.
  - In the event that something appears unclear, check with Jim

## Acceptance Criteria
- [ ] All screens match the mockups
- [ ] Document any items that are not immediately rectified 

## Next Steps
- Significant changes can go into another ticket
